changelog/fragments/anible-requierments.yaml

```diff
@@ -0,0 +1,84 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      For Ansible-based language operators, upgrade `community.kubernetes` from `1.2.1` to `2.0.1`. **IMPORTANT** The `community.kubernetes` collection is being renamed to kubernetes.core. As of version 2.0.0, the collection has been replaced by deprecated redirects for all content to kubernetes.core. If you are using FQCNs starting with `community.kubernetes`, please update them to `kubernetes.core`.
+    kind: "change"
+    breaking: true
+    migration:
+      header: For Ansible-based language operators, upgrade `community.kubernetes` from `1.2.1` to `2.0.1`.
+      body: |
+        **IMPORTANT** The `community.kubernetes` collection is being renamed to `kubernetes.core`. As of version 2.0.0, the collection has been replaced by deprecated redirects for all content to kubernetes.core. If you are using FQCNs starting with `community.kubernetes`, please update them to `kubernetes.core`.
+        Ensure that you check the [community.kubernetes changelog](https://github.com/ansible-collections/community.kubernetes/blob/main/CHANGELOG.rst) to verify if your Operator code must be updated and it is affected by the MAJOR changes.
+
+        In the `requirements.yml` file:
+
+        Replace:
+
+        ```ymal
+        collections:
+          - name: community.kubernetes
+            version: "1.2.1"
+        ```
+
+        With:
+
+        ```ymal
+        collections:
+          - name: community.kubernetes
+            version: "2.0.1"
+        ```
+  - description: >
+      For Ansible-based language operators, upgrade `kubernetes.core` from `2.2.0` to `2.3.1`. **IMPORTANT** The `community.kubernetes` collection is being renamed to kubernetes.core. As of version 2.0.0, the collection has been replaced by deprecated redirects for all content to kubernetes.core. If you are using FQCNs starting with `community.kubernetes`, please update them to `kubernetes.core`.
+    kind: "change"
+    breaking: true
```

Comment:
I don't think I'd call this a breaking change, but having it show up in the migration is convenient so sgtm

Reply:
Ops. That was a mistake +1